src/svg2pdf.js

```diff
@@ -804,16 +837,7 @@ SOFTWARE.
         pf(getAttribute(node, "height")) || (vb && vb[3]) || 0
       ];
     } else if (nodeIs(node, "g,clippath")) {
-      boundingBox = [0, 0, 0, 0];
```

Comment:
var keyword ;)

Reply:
This is not my code... It already existed at v1.5.0 and I just moved it to an external function, but I will fix it there, sure ;)

Reply:
The variable was declared earlier in the function scope:
```
var i, minX, minY, maxX, maxY, viewBox, vb, boundingBox;
```
In JavaScript `var` variables are not block-scoped, which means if a variable is declared anywhere within a function (e.g. nested in an if-block) it is available everywhere in that function.

Reply:
Yup, it's fixed, I just wanted to explain that this was a copy/paste error (what doesn't make anything better, I know...)